homeassistant/components/sensor/loopenergy.py

```diff
@@ -6,56 +6,67 @@
 """
 import logging
 
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+
 from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-from homeassistant.util import convert
 
 _LOGGER = logging.getLogger(__name__)
 
-DOMAIN = "loopenergy"
-
 REQUIREMENTS = ['pyloopenergy==0.0.14']
 
+CONFIG_ELEC = 'electricity'
```

Comment:
For the sake of consistency I would like to suggest that we use `CONF_` as prefix for configuration relevant constants. Sorry, it's a bit petty.